### PR TITLE
Fix vertical rhumb line when using PolylineGeometry

### LINKED ISSUE #8764: GeoJSON with vertical line crashes
When loading the folling Sandcastle example, it will crash with following error:

> An error occurred while rendering.  Rendering has stopped.
> undefined
> DeveloperError: Expected right to be typeof object, actual typeof was undefined
> Error
>    at DeveloperError (https://sandcastle.cesium.com/CesiumUnminified/Workers/Check-c4f3a3fc.js:62:17)
>    at Check.typeOf.object (https://sandcastle.cesium.com/CesiumUnminified/Workers/Check-c4f3a3fc.js:238:17)
>    at Cartesian3.subtract (https://sandcastle.cesium.com/CesiumUnminified/Workers/Cartesian2-a4e73c05.js:530:13)
>    at PolylineGeometry.createGeometry (https://sandcastle.cesium.com/CesiumUnminified/Workers/createPolylineGeometry.js:389:21)
>    at createPolylineGeometry (https://sandcastle.cesium.com/CesiumUnminified/Workers/createPolylineGeometry.js:523:13)
>    at createGeometry (https://sandcastle.cesium.com/CesiumUnminified/Workers/createGeometry.js:60:21)
>    at callAndWrap (https://sandcastle.cesium.com/CesiumUnminified/Workers/createTaskProcessorWorker.js:61:17)
>    at Anonymous function (https://sandcastle.cesium.com/CesiumUnminified/Workers/createTaskProcessorWorker.js:106:17)

The example just loads one simple GeoJSON Feature of type LineString. If I change any of the three coordinates slighty, it works

 ```
geojson_produces_error = {"type":"Feature","geometry":{"type":"LineString","coordinates":[[7.970063350393387,47.59408093341338,773.65],[7.970063350393387,47.59408093341338,765.55]]}};
 geojson_this_works     = {"type":"Feature","geometry":{"type":"LineString","coordinates":[[7.970063350393387,47.59408093341338,773.650001],[7.970063350393387,47.59408093341338,765.55]]}};
```

[Sandcastle example:](https://sandcastle.cesium.com/#c=xVPBbuIwEP0VKxeCFDlBIaQsKaq21a60qrQHqr00CLnJ0Hrr2Mgeg2jFv3dCugW2Paz2Up88894b+3nGa2HZWsIGLDtnGjbsEpz0Df+1z4W9ah9eGo1CarC9/qTUpZ4JXVfCoQIu6voKlsIrvDFG3Qn71SMaHfZes8zhVkl934vY0usKJWF99txWYbQUILsH89sZvVhZU3s6cAHWmvY+z2WA2xWUwZcy+AYCvaV9VAYkaADtloBjyjXdcIaWDtuzKmNsLbVAcATf3uZ8nCfJKE2zJB2n6VkeDXOejYfJWULhcECpKM9TPsrm0b+RRxnPsvl8t5t0ZuL42A4+SLfYGPvoWuwT7SRJMvgfS4cO1QLFzHhbAdl4nZDvYH6Qzas3iCsj6vDjZvbfynXDxg8VXTtC4SHuT06IT8Y0N+YvfNfVC6KgaMcLpp3kQjYrY5F5q0LOY4Rmpdr3iu989QjIK+daYRH/ERW1XDNZn9Prns55GbBKCecIWXqlZvKJejItYuKfyFrL1KCfa7BKbFvKw2B63SU550VM4XsVdj/lqOIL)

Browsers:
  Google Chrome Version 80.0.3987.163
  Firefox 75.0 (64-Bit)
  Internet Explorer 11.719.18362.0

Operating System:
  Windows 10

### PR DESCRIPTION
**Pull request by baothientran**
Fixes #8764 

The issue is that when two endpoints have the same longitude and latitude but different height, the math to calculate the number of points on rhumb line will return 0.

@lilleyse can you please take a look at it? Please let me know if I need to add anything

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @baothientran!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/fix-vertical-rhumb/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

File: Specs/Core/PolylineGeometrySpec.js [hunk 1]
```diff
@@ -184,6 +185,64 @@ describe("Core/PolylineGeometry", function () {
     expect(geometry).not.toBeDefined();
   });
 
+  it("createGeometry returns positions if their endpoints'longtitude and latitude are the same for rhumb line", function () {
+    var positions = Cartesian3.fromRadiansArrayHeights([
```
Comment (hpinkos) [inline Specs/Core/PolylineGeometrySpec.js:189]:
```suggestion
    var positions = Cartesian3.fromDegreesArrayHeights([
```

File: Specs/Core/PolylineGeometrySpec.js [hunk 2]
```diff
@@ -184,6 +185,64 @@ describe("Core/PolylineGeometry", function () {
...
+      30.0,
+      30.0,
+      5.0,
+    ]);
+    var geometry = PolylineGeometry.createGeometry(
+      new PolylineGeometry({
+        positions: positions,
+        width: 10.0,
+        vertexFormat: VertexFormat.POSITION_ONLY,
+        arcType: ArcType.GEODESIC,
```
Comment (hpinkos) [inline Specs/Core/PolylineGeometrySpec.js:202]:
```suggestion
        arcType: ArcType.RHUMB,
```

**Comment by hpinkos** [comment]
Thanks @baothientran!  I'll merge as soon as CI passes

**Comment by baothientran** [comment]
@hpinkos thank you for fixing my mistakes. I should be more careful about it in the future

**Pull request merged**